content/cpp/concepts/math-functions/terms/trunc/trunc.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'trunc()' 
+Description: 'Returns the integral part of the argument.' 
+Subjects:
+  - 'Computer Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`trunc()`** function rounds the specified argument towards zero and returns the nearest integral value not larger in magnitude than the argument. Its return value can be of [types](https://www.codecademy.com/resources/docs/cpp/data-types) `double`, `float`, or `long double`. 
+
+## Syntax
+
+```cpp
+std::trunc(x);
+```
+
+## Example
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+    double x = -3.14159;
+    double result;
+
+    result = std::trunc(x);
+
+    std::cout << "trunc(" << x << ") = " << result << "\n";
+    // Output: trunc(3.14159) = -3
+}
+```
+
```

Comment:
```suggestion

This will produce the following output:

```

And then add a newline, followed by a \```shell code block that looks like this:

```shell
// Output: trunc(3.14159) = -3
```